### ISSUE TITLE
Informative message when students leave a blank empty

### ISSUE THREAD

**Issue by machow** [datacamp/pythonwhat#211]
That is, submitting `x = ____` would lead to a feedback message like, "Be sure to fill in all of the blanks (`____`)."

**Comment by martijnsublime** [comment]
Issue moved to [datacamp/learn-features #1](https://github.com/datacamp/learn-features/issues/1) via [**ZenHub**](https://www.zenhub.com/)

**Comment by filipsch** [comment]
@martijnsublime isn't this part of the feedback systems that I would be working on again soon, i.e. Content Engineering?

**Comment by martijnsublime** [comment]
Yes. I started on Friday working through this and I started to realize there is a lot of stuff in there that will end up on your plate.

**Comment by filipsch** [comment]
I would keep this stuff inside `pythonwhat` then, so I don't lose track of it.
